doc/ref_cert/RC2/chapters/chapter02.md

```diff
@@ -313,12 +313,12 @@ The following software versions are considered to benchmark Kubernetes v1.21
 
 ### Dataplane benchmarking
 
-[Kubernetes perf-tests](https://github.com/kubernetes/perf-tests) stores
-various Kubernetes-related performance test related tools especially
+[Kubernetes perf-tests repository](https://github.com/kubernetes/perf-tests)
+stores various Kubernetes-related performance test related tools especially
```

Comment:
Can we change to "hosts various Kubernetes-related performance test related tools especially"